app/controllers/api/v1/users_controller.rb

```diff
@@ -0,0 +1,35 @@
+class Api::V1::UsersController < ApplicationController
+  def create
+    user = User.new(user_params)
+
+    if User.exists?(email: user.email)
+      render json: email_error(user), status: :unprocessable_entity
+    elsif user.password != user.password_confirmation
+      render json: password_error(user), status: :unprocessable_entity
+    else user.save
+      render json: UserSerializer.new(user), status: :created
+    end
+  end
+
+
+
+  private
+
+  def user_params
+    params.require(:user).permit(:name, :email, :password, :password_confirmation)
+  end
+
+  def password_error(user)
+    {
```

Comment:
Got two simple error handling methods that could be refactored in to the error serializer, but work for now.